### DISCUSSION TITLE
I have 4 pull requests approved, but got achievement only for one, it's so sad..

### DISCUSSION THREAD

**Discussion by emilycodestar** [community/community#53980] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I have 4 pull requests approved, but got achievement only for one, it's so sad..

**Comment by drippy-cat** [comment, marked as answer]
Which achievement is this referring to? According to [this](https://github.com/drknzz/GitHub-Achievements) unofficial list of badges, you need **at least opened a pull request that has been merged sixteen times** (for a bronze pull shark badge) or **coauthored commits on merged pull request at least ten times** (for a bronze pair extraordinaire badge). Hope this helps!

**Comment by YasinDehfuli** [comment]
You Can use This Step-By-Step Course For any badges you want to get : 
in achievement list, click how to get button and learn how to get github achievements step by step by images.
Link:
https://github.com/4xmen/Get-Github-Achievements

**Comment by emilycodestar** [comment]
Thanks everyone! It was really helpful for me
